pkgs/by-name/op/openmsx/package.nix

```
@@ -32,6 +33,14 @@ stdenv.mkDerivation (finalAttrs: {
     fetchSubmodules = true;
   };
 
+  patches = [
+    (fetchpatch {
```

```suggestion
    (fetchpatch2 {
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches

---

That's good to know. I was reading https://github.com/NixOS/nixpkgs/blob/master/doc/build-helpers/fetchers.chapter.md instead.